### ISSUE TITLE
Unable to add graphite functions in the latest build

### ISSUE THREAD

**Issue by stahor** [grafana/grafana#11434]
Hi there. I faced quite weird issue while was trying to upgrade grafana. It's not possible to add functions to the query (there is no button for it). Chrome console contains the following error message:
```
Error: [$injector:unpr] Unknown provider: tProvider <- t <- graphiteAddFuncDirective
```
It's still possible to add function manually in edit mode so the issue is not very critical for me. The strangest thing in this situation is that I can't reproduce the issue in development mode. I can reproduce it only with grafana-server launched.
My current branch is at b41370ca498816f444363ed7511dced638dc54f6

**Comment by daniellee** [comment]
We missed adding an `/** @ngInject */` when converting this file from JavaScript to TypeScript. Angular injection works fine without it in development but not in the production build.

**Comment by stahor** [comment]
Yea I haven't noticed. It's also missing on `graphiteFuncEditor`. 
I can make a pull request with these 2 but there is still a chance that some other parts of system are not annotated. So maybe it makes sense to test the system in production mode somehow and fix all issues at once.

**Comment by torkelo** [comment]
Was confused why this bug was introduced in stable release as it seemed like a strange PR to cherry pick to stable branch, and indeed the PR was never cherry picked. So this does not need to be in v5.0.5 milestone but in v5.1 

The js file is still there in stable branch:
https://github.com/grafana/grafana/blob/v5.0.x/public/app/plugins/datasource/graphite/add_graphite_func.js 

add /** @ngInject */ above this line
https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/graphite/add_graphite_func.ts#L7

**Comment by stahor** [comment]
@torkelo annotation also has to be added above this line for function editor does not work as well. And it is not so easy to find out what else is broken.
https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/graphite/func_editor.ts#L6

**Comment by daniellee** [comment]
@torkelo yeah, noticed today that it wasn't included in 5.0.4. <breathes sigh of relief>

Will fix this for 5.1 and double check the other js files that were converted to ts.
